rules/windows/image_load/sysmon_uac_bypass_via_dism.yml

```diff
@@ -0,0 +1,28 @@
+title: UAC Bypass With Fake DLL
+id: a5ea83a7-05a5-44c1-be2e-addccbbd8c03
+status: experimental
+description: Attempts to load dismcore.dll after dropping it.
+references:
+    - https://steemit.com/utopian-io/@ah101/uac-bypassing-utility
+tags:
+    - attack.persistence
+    - attack.defense_evasion
+    - attack.privilege_escalation
+    - attack.t1548.002
+    - attack.t1574.002
+author: oscd.community, Dmitry Uchakin
+date: 2020/10/06
+logsource:               
+    category: image_load 
+    product: windows     
+detection:
+    selection:
+        Image:
+            - '\dism.exe'
+        ImageLoaded:
+            - '\dismcore.dll'
```

Comment:
could you please apply `endswith` [value modifier](https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#value-modifiers) here?

Reply:
I think that if you add a filter that excludes:
	`ImageLoaded:  C:\Windows\System32\Dism\dismcore.dll`
then the number of false positives will decrease. Just in case, please check that there are no false negative.

Reply:
thanks!